packages/rest-typings/src/v1/channels/ChannelsConvertToTeamProps.ts

```diff
@@ -22,6 +22,15 @@ const channelsConvertToTeamPropsSchema = {
 			required: ['channelName'],
 			additionalProperties: false,
 		},
+		{
+			type: 'object',
+			properties: {
+				channelId: { type: 'string' },
+				channelName: { type: 'string' },
+			},
+			required: ['channelId', 'channelName'],
+			additionalProperties: false,
+		},
```

Comment:
this is wrong. this will allow passing BOTH at the same time.

Reply:
to add: this doesn't make sense because 
1. passing the id and the name of the same room is redundant
2. passing the id of one room and name of another .. endpoint at its current stage will just process the id, so the name is unnecessary. and we definitely don't want to process each item individually, that's just bad design.

Reply:
This has been requested by the mobile team, but I've started a conversation with the architecture team and we decided to remove this change. Thanks for pointing this!